rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :preson
```

Comment:
- [ ] Kindly correct your typo error here. I think this should be `:person`, not `:preson`